### PR TITLE
 #23 Eventpage component

### PR DESCRIPTION
**Pull request by tasneembhiri**
- create eventpage 
  bring event title and event date from database (events frontend-backend)
  add style for eventpage (mobile first)

### CODE REVIEW

File: client/src/components/Layout/Events/Events.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+  render () {
+    const { events } = this.state
+    return (
+      <div className="events">
+
+        {
+          events.map(event => {
+            return (
+
```
Comment (yaqootturman) [inline client/src/components/Layout/Events/Events.js]: Delete the empty line

File: client/src/components/Layout/Events/Events.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  render () {
+    const { events } = this.state
+    return (
+      <div className="events">
+
+        {
+          events.map(event => {
+            return (
+             
+              <div className="event">
```
Comment (yaqootturman) [inline client/src/components/Layout/Events/Events.js]: Give each 'event' class more width and add a media query for bigger screens

File: client/src/components/Layout/Events/Events.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react'
+import axios from 'axios'
+import './Events.css'
```
Comment (yaqootturman) [inline client/src/components/Layout/Events/Events.js]: The CSS files should not be capital

**Review by yaqootturman — CHANGES_REQUESTED** [review]

**Review by yaqootturman — COMMENTED** [review]
Looks good, fix the conflict and let other members see it

**Review by yaqootturman — APPROVED** [review]

**Review by bayanseder — APPROVED** [review]

**Review by Hanan795 — APPROVED** [review]

**Pull request merged**